xbar/plugins/notifications.30s.py

```diff
@@ -140,8 +140,8 @@ def format_notification( notification ):
     if latest_comment_url:
         formatted['href'] = ( make_github_request( latest_comment_url ) or {} ).get( 'html_url', formatted['href'] )
     # Try to hack a web-viewable URL if the last check failed
-    if formatted['href']:
-        formatted['href'] = re.sub( 'api\.|api/v3/|repos/', '', re.sub( '(pull|commit)s', ur'\1', formatted['href'] ) )
+    # if formatted['href']:
+    #     formatted['href'] = re.sub( 'api\.|api/v3/|repos/', '', re.sub( '(pull|commit)s', ur'\1', formatted['href'] ) )
```

Comment:
There is a syntax error here. I lazily couldn't figure it out, but also am not sure if this is needed to work.

Reply:
Let's just remove it.

Reply:
Heh, seems I use this feature. Tapping a row now brings me to a json response instead of the actual destination of the notification.